source/clear-linux/get-started/bare-metal-install-server/bare-metal-install-server.rst

```diff
@@ -241,107 +241,115 @@ on it, and accept the `Default partition schema`_.
 Advanced Configuration
 ----------------------
 
-Use this method to manually configure partitions. These must meet
-`Default partition schema`_. You may also choose `Disk encryption`_ during
-configuration of each partition.
+Use this method to manually configure partitions. Our example follows the
+`Default partition schema`_. Use these defaults, except note that the size
```

Comment:
The **Default partition schema** indicates `root` as a FileSystem. Instead it should be `ext4`.
Also maybe @mdhorn can confirm if the default **swap** size is `32M` or `250M`

Reply:
There are two ways to solve this.  Tell me your preference. I prefer 2.

1. Change `VFAT` to `boot` (assuming this is just a label).

2. Change all of them to the type of FileSystem as follows: 

- FileSystem:  `boot --> VFAT(FAT32)`
- FileSystem:  `swap --> linux-swap`
- FileSystem:  `root --> ext[234] or XFS`

Reply:
I prefer 2

Reply:
I'll revise this table , `Default partition schema`, per my 2 above in both live-server and the live-desktop docs. @mdhorn if you object, please say so now.